cmdstanpy/model.py

```diff
@@ -505,3 +506,56 @@ def _do_sample(self, stanfit: StanFit, idx: int) -> None:
                 transcript.write('ERROR')
                 transcript.write(stderr.decode('utf-8'))
         stanfit._set_retcode(idx, proc.returncode)
+
+    def run_generated_quantities(
```

Comment:
this command needs to take a StanFit object as input.
it returns a StanFit object which contains the generated quantities.